pathfinding_service.rst

```diff
@@ -247,36 +113,6 @@ Example
         "value": 45,
         "max_paths": 10
     }'  /api/1/paths
-    // Request with specific preference
```

Comment:
I like the curl examples and error output. Why don't we keep the parts that still apply?

Reply:
reintegrated the curl examples and the errors output without fees